plasma_cash/utils/merkle/sparse_merkle_tree.py

```diff
@@ -0,0 +1,37 @@
+from collections import OrderedDict
+from ethereum.utils import sha3
+
+
+class SparseMerkleTree(object):
+
+    def __init__(self, leaves={}):
+        self.depth = 256
+        self.leaves = OrderedDict(sorted(leaves.items(), key=lambda t: t[0]))
+        self.default_nodes = [b'\x00' * 32]
+        self.create_default_nodes()
+        if bool(leaves):
+            self.root = self.create_tree()
+        else:
+            self.root = self.default_nodes[self.depth - 1]
+
+    def create_default_nodes(self):
+        for i in range(1, self.depth):
```

Comment:
Minor: could rename `i` to `level` for better readability

Reply:
Great